gradle/README.md

```diff
@@ -1,9 +1,9 @@
 # How to upgrade the Gradle version
 
 Visit the [Gradle website](https://gradle.org/releases/) and decide the:
-*  desired version
-*  desired distribution type
-*  what is the sha256 for the version and type chosen
+*   desired version
+*   desired distribution type
+*   what is the sha256 for the version and type chosen
```

Comment:
Nit: this double-spacing after the bullet point is pretty unconventional. Usual syntax throughout most of the codebase is:
```
 - first bullet point
 - second bullet point
```

Note that more than 75% use dashes vs asterisks as well.

Run `find . -name '*.md' | xargs grep '^ \?[-*] '` for details.

Reply:
Thanks, wasn't aware of that.

I tried it, but Codacy still rejects it. Probably some temporary rule misconfiguration.

Reply:
Just check and saw that Codacy doesn't like the extra space before the `-`. I just scanned through our codebase how we did it so far and just disabled this rule.